### ISSUE TITLE
Performance improvement for fetching missing service_aups

### ISSUE THREAD

**Issue by oharsta** [SURFscz/SBS#1559]

**Comment by oharsta** [comment]
@baszoetekouw Can you test for me the performance of the login of a user with many services connected to users CO's after deletion of the records of that user in the `service_aups` table?

**Comment by baszoetekouw** [comment]
@mrvanes could you try to do this test?  Or even better, adjust the demo seed to add the required user and services?

**Comment by baszoetekouw** [comment]
Yes, and I think it makes sense to check this (and have a way to validate any optimizations).  However, I don't think there is any usable test-user available, so we need to add one in the test seed.

**Comment by logan-life** [comment]
Blocked until https://github.com/SURFscz/SBS/issues/1567 is done

**Comment by logan-life** [comment]
@HarryKodden after you finish the stress seed thing, please check this, time the fetch stage of the load, and put the numbers in this ticket.

Then we decide if the performance is in need of improvement (or not). If yes, Okke has solution ready. If not, this ticket can be closed.
